public/js/common/export/export.js

```diff
@@ -37,8 +37,10 @@ export var ExportModal = React.createClass({
     },
 
     getGraphImage: function() {
+        var JsonLocalStorageObj = JSON.stringify(localStorage);
```

Comment:
It isn't good to send the *entire* local storage as JSON to the server. Instead, for all these requests you should only extract the key-value pairs that are necessary for that particular request. This should have the added benefit of making the server code a little simpler.